sanowret-crystal.lic

```diff
@@ -45,17 +46,17 @@ class SanowretCrystal
     if @worn_crystal
       use_crystal
     else
-      case DRC.bput('tap my sanowret crystal', /^You tap (?:a|an) .*sanowret crystal inside your .*.$/, /^You tap (?:a|an) .*sanowret crystal.* that you are wearing.$/, /^You tap (?:a|an) .*sanowret crystal that you are holding.$/, 'I could not find what you were referring to.', 'Not here, ', 'a wave of Corruption magic')
+      case DRC.bput("tap my #{@noun} crystal", /^You tap (?:a|an) .*sanowret crystal inside your .*.$/, /^You tap (?:a|an) .*sanowret crystal.* that you are wearing.$/, /^You tap (?:a|an) .*sanowret crystal that you are holding.$/, 'I could not find what you were referring to.', 'Not here, ', 'a wave of Corruption magic')
       when /You tap (?:a|an) .*sanowret crystal inside your/
-        DRC.bput('get my sanowret crystal', 'You get', 'What were you referring to')
+        DRC.bput("get my #{@noun} crystal", 'You get', 'What were you referring to')
         use_crystal
-        DRC.bput('stow my sanowret crystal', 'You put', 'Stow what')
+        DRC.bput("stow my #{@noun} crystal", 'You put', 'Stow what')
       when /^You tap (?:a|an) .*sanowret crystal.* that you are wearing\.$/
```

Comment:
❓ Does this line (and lines 50, 57, etc.) need to be updated to use the noun instead of `sanowret`?

Reply:
Yes.  I'm not sure how many of the other sanowret crystals have this issue, but the hoarfrost white sanowret crystal uses white as the adjective.

I really must have been tired, I should have named the variable adjective not noun.

Reply:
The match lines don't need to be updated because they have a .* in front of sanowret, so it'll match any adjective, or none.